docs/frontend/configuring-image-sizes-for-individual-devices-width.md

```diff
@@ -1,82 +0,0 @@
-# Configuring Image Sizes for Individual Devices Width
-
-In Shopsys Platform you can configure image sizes for individual devices width. This is allowed by HTML element [Picture element (see more in Mozilla official documentation)](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/picture).
-
-## Introduction
```

Comment:
this file have to be removed from the docs/frontend/navigation.yml, or the documentation will not build. (see failed container locally)

Reply:
ah, thanks! 41f16c887